lib/puppet/provider/scheduled_task/win32_taskscheduler.rb

```diff
@@ -101,6 +100,8 @@ def trigger
       puppet_trigger['start_date'] = self.class.normalized_date("#{trigger['start_year']}-#{trigger['start_month']}-#{trigger['start_day']}")
       puppet_trigger['start_time'] = self.class.normalized_time("#{trigger['start_hour']}:#{trigger['start_minute']}")
       puppet_trigger['enabled']    = trigger['flags'] & Win32::TaskScheduler::TASK_TRIGGER_FLAG_DISABLED == 0
+      puppet_trigger['minutes_interval'] = trigger['minutes_interval'] ||= 0
+      puppet_trigger['minutes_duration'] = trigger['minutes_duration'] ||= 0
```

Comment:
If `duration` defaults to 0, then to run a task every 30 minutes, they have to specify:

```
scheduled_task { 'something': 
  command => '...'.
  minutes_interval => 30,
  minutes_duration => something bigger than 30
}
```

The duration is really a window of time, the "for a duration of" in https://github.com/puppetlabs/puppet/pull/3637#discussion_r27248447.

I'm thinking the default should instead be the number of minutes in the day, so that you can simply do:

```
scheduled_task { 'something': 
  command => '...'.
  minutes_interval => 30,
}
```

and it works as expected, repeats every 30 minutes.